web/src/components/PoiListItem.tsx

```diff
@@ -74,16 +51,27 @@ const PoiListItem = ({ poi, distance, selectPoi }: PoiListItemProps): ReactEleme
 
   return (
     <>
-      <ListItemContainer id={slug}>
-        <LinkContainer onClick={selectPoi} tabIndex={0} label={title}>
-          <Thumbnail alt='' src={thumbnail || PoiThumbnailPlaceholder} />
-          <Description>
-            <Title>{title}</Title>
-            {distance !== null && <Distance>{t('distanceKilometre', { distance: distance.toFixed(1) })}</Distance>}
-            <Category>{category.name}</Category>
-          </Description>
-        </LinkContainer>
-      </ListItemContainer>
+      <ListItem disablePadding>
+        <StyledListItemButton onClick={selectPoi} id={slug} to={slug} component={Link} aria-label={title}>
+          <StyledListItemAvatar>
+            <Avatar src={thumbnail || PoiThumbnailPlaceholder} alt='' variant='square' />
+          </StyledListItemAvatar>
+          <StyledListItemText
+            disableTypography
+            primary={
+              <StyledTypography variant='title2' component='h2'>
```

Comment:
What matches the design is `body2` (fontSize: 14, fontWeight: 400)
```suggestion
              <StyledTypography variant='body2' component='h2'>
```